apps/api/src/app/workflows-v2/usecases/patch-workflow/patch-workflow.usecase.ts

```diff
@@ -11,18 +15,35 @@ export class PatchWorkflowUsecase {
   constructor(
     private getWorkflowWithPreferencesUseCase: GetWorkflowWithPreferencesUseCase,
     private notificationTemplateRepository: NotificationTemplateRepository,
-    private getWorkflowUseCase: GetWorkflowUseCase
+    private getWorkflowUseCase: GetWorkflowUseCase,
+    @Optional()
+    private sendWebhookMessage: SendWebhookMessage
```

Comment:
i never saw such a pattern in the constructor. what does it do?

Edit:
i see why you did this, but won't lie I don't like that you need to guess that it is possible undefined :( 

```suggestion
    @Optional()
    private sendWebhookMessage?: SendWebhookMessage
```